components/Navbar.vue

```diff
@@ -39,9 +39,12 @@
                 ref="mobilSearchRef"
                 hide-filter
                 class="is-flex-grow-1 mt-3" />
-              <button class="cancel-btn p-3" @click="hideMobileSearchBar">
+              <NeoButton
+                variant="text"
+                class="cancel-btn p-3 mt-3 is-shadowless"
```

Comment:
We could use`no-border` `is-capitalized` instead of `cancel-btn`

<img width="276" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/a070f43e-7ed4-4a8a-86c5-b4dfbcd4546f">

Reply:
(& `is-clickable`)